test/logger_json/formatters/google_cloud_test.exs

```diff
@@ -450,4 +450,14 @@ defmodule LoggerJSON.Formatters.GoogleCloudTest do
              "serviceContext" => %{"service" => "nonode@nohost"}
            } = log_entry
   end
+
+  test "passing options to encoder" do
+    formatter = {GoogleCloud, encoder_opts: [pretty: true]}
+    :logger.update_handler_config(:default, :formatter, formatter)
+
+    assert capture_log(fn ->
+             Logger.debug("Hello")
+           end) =~
+             ~r/^{\n\s{2}"message": "Hello",\n\s{2}/
```

Comment:
Maybe it would be enough to just check for newline and spaces before the `"message"`?
```suggestion
             ~r/\n\s{2}"message": "Hello"/
```

Reply:
That works for me. I'll update the PR.

Reply:
Resolved by [0e2fa6b](https://github.com/Nebo15/logger_json/pull/130/commits/0e2fa6bc070d7ec2c710b84ce00049d8b503692f)